src/types.ts

```diff
@@ -1,11 +1,19 @@
 export interface File {
-  name: string,
+  name: string
   data: Buffer
-  size: number,
-  encoding: string,
-  tempFilePath: string,
-  truncated: boolean,
-  mimetype: string,
-  md5: string,
-  mv: Function
+  size: number
+  encoding: string
+  tempFilePath: string
+  truncated: boolean
+  mimetype: string
+  md5: string
+  mv: () => undefined
+}
+
+export interface Files {
+  [file: string]: File
+}
+
+export interface UploadParams {
```

Comment:
put those two right besides the api handler, or maybe inline them

Reply:
It is used in nginx file, i don't think it is good idea, i remember out talk from yesterday, but i think it is the best option